build_debian.sh

```diff
@@ -413,6 +413,20 @@ sudo LANG=c chroot $FILESYSTEM_ROOT chmod 644 /etc/group
 sudo LANG=C chroot $FILESYSTEM_ROOT /bin/bash -c "mkdir -p /etc/initramfs-tools/conf.d"
 sudo LANG=C chroot $FILESYSTEM_ROOT /bin/bash -c "echo 'MODULES=most' >> /etc/initramfs-tools/conf.d/driver-policy"
 
+# Ensure the relevant directories exist
+sudo mkdir -p /etc/initramfs-tools/scripts/init-premount
+sudo mkdir -p /etc/initramfs-tools/hooks
+
+# Copy the network setup scriptgit
```

Comment:
I'm not sure why we need this change, as long as SSH works to the remote destination, it should be enough, right?

Reply:
What will be impact for DUMP remote mode feature if we dont have this pull request?

Reply:
The goal here is to ensure the "eth0" interface remains active, as this is essential for the smooth operation of the SSH remote feature. It's not just about configuring SSH parameters in the file; once the feature values are set, we also need to bring up the "eth0" interface. Without doing so, we encounter a "network unreachable" error. Enabling the "eth0" interface requires making all these changes, and each step is crucial for its activation